packages/main/test/pages/Table.html

```diff
@@ -27,9 +27,9 @@
 		<ui5-table-growing id="growing" type="Scroll" slot="features"></ui5-table-growing>
 		<ui5-table-selection id="selection" selected="0 2" slot="features"></ui5-table-selection>
 		<ui5-table-header-row slot="headerRow" sticky>
-			<ui5-table-header-cell id="produtCol" width="300px"><span>Product</span></ui5-table-header-cell>
-			<ui5-table-header-cell id="supplierCol" width="400px">Supplier</ui5-table-header-cell>
-			<ui5-table-header-cell id="dimensionsCol" importance="-1" min-width="300px">Dimensions</ui5-table-header-cell>
+			<ui5-table-header-cell id="produtCol" h-Align="Right" width="300px"><span>Product</span></ui5-table-header-cell>
```

Comment:
What happens if I write something like `h-Align="stretch"`?

Reply:
All values for `justify-content` can be used right now (see: [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content).